helm/defectdojo/templates/django-deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- $revisionHistoryLimit := or .Values.django.revisionHistoryLimit .Values.revisionHistoryLimit }}
```

Comment:
Hi, so maybe I wasn't clear in my proposal about the simplification. I was thinking just to have one definition like `.Values.revisionHistoryLimit` and ommit under specific deployments in values.  Like is in the e.g . https://github.com/bitnami/charts/blob/main/bitnami/apache/values.yaml .

Reply:
I think this is actually an even better solution than just exposing a single value. If you want to set a standard value for all deployments it works exactly as you would want, while also providing more flexibility to users that want to specify different limits for different parts of the overall deployments 🤔

Reply:
It is more flexible but also more complex. If there is a use case for it, the flexible one is ok. Personally can't imagine a relevant use case for this. I mean it is just the revision history and not nothing imported to customize per deployment.

Reply:
fwiw i left the conditional rendering (but not the conditional assignment) in so that users who did not specify this value  in any other customizations. This will keep any templates from being unexpectedly modified by an opinionated statement of the default history limit by these templates and defer to Kubernetes's defaults. Hope that is okay.